personalwebsite/projects/templates/projects.html

```diff
@@ -7,7 +7,7 @@ <h1 class="display-5 font-family-roboto">Projects</h1>
 </div>
 {% for project in all_projects %}
 <div class="row">
-    <div class="col-lg-4 col-md-5 col-sm-6 text-center my-3">
+    <div class="col-lg-4 col-md-12 col-sm-12 text-center my-3">
```

Comment:
Line 10:
`<div class="col-lg-4 col-md-12 col-sm-12 text-center my-3">`

Might be better to remove text-center from main div container:
` <div class="col-lg-4 col-md-12 col-sm-12 my-3">`
and add it to only to text that need it, otherwise all of the text inherits from it.

Reply:
That text center class was actually intentional to center not only the text in that block but also the cover image when the window was smaller. However, I'm going to change this and put the title over the description as suggested earlier.